util/util.go

```diff
@@ -244,12 +244,25 @@ func SelectDuration(opt, def time.Duration) time.Duration {
 	return opt
 }
 
-// Min returns min(a,b)
-func Min(a, b int) int {
-	if a < b {
-		return a
+// SelectBool takes an option and a default value and returns the default value if
+// the option is equal to zero, and the option otherwise.
```

Comment:
I think it is good to not call it false here, even though that is what the change is, but I think we should refer to it as the [zero value](https://golang.org/ref/spec#The_zero_value)